### ISSUE TITLE
The power supply status file contains "Unknown"

### ISSUE THREAD

**Issue by Sichroteph** [Ninoh-FOX/RG280V_ROGUE#6]
The file "status" located in sys/class/power_supply/battery always contain the string "Unknown"

The issue is that Retroarch is not able to display the battery percentage even if the file "capacity" contains the actual battery remaining.

In order for retroarch to work, the file "status" needs to contain "Discharging", any idea how I can do this ? This file seems locked by the system (DinguxCmdr can't overwrite it)

Edit : I mounted the rootfs file on ubuntu but the sys folder contains nothing.

**Comment by Ninoh-FOX** [comment]
This is strange because I see the battery icon in retroarch dingux version (in dingux beta version RA , the battery not work in stock)

**Comment by Sichroteph** [comment]
You are right. I may have downloaded the beta version for the rogue firmware and assumed it was because of this file. 
It is fine on the non beta retroarch. I tried to install the stock firmware too, and the status file contains "Unknown" too. The link I provided put me in the wrong direction. Thanks a lot for solving my issue. 

On another field, I noticed that the frontend_start script doesn't work on the stock firmware (Even by launching it directly) but does fine on the Rogue.
